src/translator/translator.h

```diff
@@ -226,7 +241,6 @@ class TranslateService : public ModelServiceTask {
       auto precison = options_->get<std::vector<std::string>>("precision", {"float32"});
       graph->setDefaultElementType(typeFromString(precison[0])); // only use first type, used for parameter type in graph
       graph->setDevice(device);
-      graph->getBackend()->configureDevice(options_);
```

Comment:
What happened here? (This change might need manual propogation into bergamot-translator repo, hence asking).

Reply:
I guess this part is updated by marian-nmt/marian-dev. A mistake I made during the rebase. I will fix this. Many thanks for pointing out!